- name: "Default Zone"
  remove_surface: true
  parent_frame_id: "base_link"
  child_frame_id: "camera_depth_frame"
  bounding_frame_id: "base_link"
  segmentation_frame_id: "base_link"
  z_min: 0.10
